style.css

```diff
@@ -1 +1,314 @@
 /*  Add your styling here */
+html {
+  margin: 0 auto;
+  padding: 0;
+  font-family: "Lora", serif;
+  background: linear-gradient(
+    307deg,
+    rgba(255, 200, 221, 1) 35%,
+    rgba(189, 224, 254, 1) 100%
+  );
+}
+
+* {
+  box-sizing: border-box;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*intro*/
+.intro {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.intro-img {
+  width: 50%;
+  padding: 0 1rem;
+}
+
+.intro-img-src {
+  width: 8rem;
+  height: 8rem;
+}
+
+.intro-text {
+  display: flex;
+  padding: 0 2rem 1rem 0;
+  flex: 50%;
+  justify-content: flex-end;
+  align-items: flex-end;
+  font-size: 70%;
+}
+
+@media only screen and (min-width: 540px) {
+  .intro-text {
+    padding-bottom: 1rem;
+    font-size: 100%;
+  }
+}
+
+@media only screen and (min-width: 541px) {
+  .intro-text {
+    padding-bottom: 1rem;
+    font-size: 100%;
+  }
+}
+
+/*nav*/
+.header {
+  background-color: #cdb4db;
+  border: 1px solid white;
+  padding: 0rem 1rem;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0rem 0.5rem;
+}
+
+.hamburger {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 1.5rem;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #101010;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.nav-item {
+  margin-left: 5rem;
```

Comment:
this will always give the items a margin-left, even on smaller screens, causing them to overlap